subprojects/file-watching/src/main/java/org/gradle/internal/watch/registry/FileWatcherUpdater.java

```diff
@@ -25,27 +25,28 @@
 
 public interface FileWatcherUpdater extends SnapshotHierarchy.SnapshotDiffListener {
```

Comment:
I still think this guy shouldn't implement the listener, but instead be called by a listener implementation.

Reply:
It now doesn't implement the listener any more.